src/components/ButtonCreateList/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
We consider that file is **necessary** because I need to get a `token `from` lib/tokens.js` and store it in `LocalStorage`. We think that there is a little confusion because according to the image provided there should be **_two buttons_** . 

![image](https://user-images.githubusercontent.com/8689897/104786912-b4681c00-5764-11eb-89eb-0b9717285320.png)

Maybe, we are wrong, please, could you help us with this confusion or any clarification? 🙏